spinn_front_end_common/interface/interface_functions/insert_extra_monitor_vertices_to_graphs.py

```diff
@@ -144,7 +145,8 @@ def _handle_data_extraction_vertices(
                     machine_vertex = equiv_vertex.machine_vertex
             else:
                 machine_vertex = self._exists_equiv_vertex(
-                    ethernet_chip.x, ethernet_chip.y, machine_graph,
+                    ethernet_chip.x, ethernet_chip.y,
+                    vertex_to_ethernet_connected_chip_mapping,
                     DataSpeedUpPacketGather)
```

Comment:
remove as not need type
```suggestion
                    )
```